lib/Twig/ExtensionInterface.php

```diff
@@ -28,35 +28,35 @@ public function initRuntime(Twig_Environment $environment);
     /**
      * Returns the token parser instances to add to the existing list.
      *
-     * @return Twig_TokenParserInterface[]
+     * @return Twig_TokenParserInterface[]|Twig_TokenParserInterface
      */
     public function getTokenParsers();
 
     /**
      * Returns the node visitor instances to add to the existing list.
      *
-     * @return Twig_NodeVisitorInterface[] An array of Twig_NodeVisitorInterface instances
+     * @return Twig_NodeVisitorInterface[]|Twig_NodeVisitorInterface An array of Twig_NodeVisitorInterface instances
```

Comment:
changing the signature of an interface is actually a BC break for consumers of the interface.

and I know of some usages of these methods outside of the Twig_Environment, so this cannot be done in 1.x